client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -90,6 +103,7 @@
     <template v-else>
       <dp-data-table
         class="overflow-x-auto"
+        :class="{ 'px-4 overflow-y-scroll h-[85%]': isFullscreen }"
```

Comment:
Again: why do we need a percentual height here? is it an issue with the header not being fixed when in fullscreen mode correctly?

Reply:
to make the table scrollbar, I have to add a height to the DataTable container. Otherwise, when the scrollbar is adjusted to the main container, the arrow will be hidden behind the header.

`overflow-y` is set to 'table' (in this case, I have to set the height of the table)
![image](https://github.com/demos-europe/demosplan-core/assets/110987766/a88b2ab2-e801-400a-864f-d7240b3cb99b)

`overflow-y` is set to main container (in this case the sticky header should be adjusted)
![image](https://github.com/demos-europe/demosplan-core/assets/110987766/2a94551b-d49d-43eb-a1e2-ad5a0a80ff11)